BParkingNano/python/tracksBPark_cff.py

```diff
@@ -44,7 +44,7 @@
         isMatchedToSoftMuon = Var("userInt('isMatchedToSoftMuon')",bool,doc="track was used to build a muon passing softID", precision=10),
         isMatchedToMediumMuon = Var("userInt('isMatchedToMediumMuon')",bool,doc="track was used to build a muon passing mediumID", precision=10),
         isMatchedToEle = Var("userInt('isMatchedToEle')",bool,doc="track was used to build a PF ele", precision=10),
-        nValidHits = Var("userInt('nValidHits')", int,doc="Number of valid hits on track", precision=10),
+        #nValidHits = Var("userInt('nValidHits')", int,doc="Number of valid hits on track", precision=10),
```

Comment:
this was recently added in PR69, so I would keep it

Reply:
changed